vsintegration/src/FSharp.LanguageService.Base/FSharp.LanguageService.Base.csproj

```diff
@@ -23,7 +24,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <EmbeddedResource Update="Microsoft.VisualStudio.Package.LanguageService.resx">
+    <EmbeddedResource Include="Microsoft.VisualStudio.Package.LanguageService.resx">
```

Comment:
Same `Update` vs. `Include` comment as above.

Reply:
As mentioned below, the default globbing was incorrectly picking up resources from a project in `MockTypeProviders` subfolder after you built those projects